src/main/java/org/javarosa/core/util/externalizable/ExtWrapArrayListMultiMap.java

```diff
@@ -0,0 +1,93 @@
+package org.javarosa.core.util.externalizable;
+
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.Multimap;
+
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.util.List;
+
+public class ExtWrapArrayListMultiMap extends ExternalizableWrapper {
```

Comment:
For simplicity I'd probably just call this class `ExtWrapMultiMap`  and see the below